packages/node_modules/@webex/plugin-meetings/src/members/index.js

```diff
@@ -71,23 +108,24 @@ export default class Members extends StatelessSparkPlugin {
    * @returns {Object}
    */
   handleLocusInfoUpdatedParticipants(payload) {
-    const membersUpdate = this.membersCollection.update(payload);
-    if (membersUpdate) {
-      this.trigger('MEMBERS_UPDATE', membersUpdate);
-    }
-    return membersUpdate;
+    console.log(payload);
+    // const membersUpdate = this.membersCollection.update(payload);
```

Comment:
Remove?

Reply:
ah i was leaving that as a placeholder for delta